app/controllers/concerns/multiple_destroy.rb

```diff
@@ -11,7 +11,7 @@ def multiple_destroy
       job_params = {
         author_email: current_user.email,
         ids: params[:ids],
-        klass: params[:controller].singularize.capitalize,
+        klass: params[:controller].split('/')[-1].singularize.capitalize,
```

Comment:
I thought we must have done this before, so I opened /controllers/concerns/ looking for instances, and found:

https://github.com/dradis/dradis-ce/blob/be46be1fff57c45c87a42a1956c3ded4191bf335/app/controllers/concerns/mentioned.rb#L15

so it seems that `params[:controller].split('/')[-1]` can be turned into [controller_name](https://api.rubyonrails.org/classes/ActionController/Metal.html#method-c-controller_name)?